process/charter.html

```diff
@@ -178,13 +178,16 @@ <h2>3. <a id="creation" name="creation">Charter Creation</a></h2>
 
 <h3 id="horizontal-review">3.1 Horizontal Review</h3>
 
-	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:
-	<ol><li>Opening a <a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">Chartering issue</a> in the <a href="https://github.com/w3c/strategy/issues">Strategy Repo</a>,</li>
+	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:</p>
+	<ol>
+  <li>Moving/Copying the draft charter into the <a href="https://github.com/w3c/charter-drafts/">charter-drafts Repo</a>, naming it using the convention [YYYY]/[shortname-name]-[group-type].html .</li>
```

Comment:
I think it should be OK for people to do the drafting work in the chartering repo. It's also OK if people want to start elsewhere, but having a consistent repo to host a given charter as it is revised is helpful, and to the extent people want to do it, it seams reasonable to allow it.

Triggering horizontal review on the draft should be done by filing an issue requesting review, not by putting the file in the repo.